src/main/java/seedu/address/logic/LogicToDoManager.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+
+import seedu.address.commons.core.ComponentManager;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.CommandToDo;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.ToDoListParser;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ModelToDo;
+import seedu.address.model.todolist.ToDoListEvent;
+
+/**
+ * The main LogicToDoManager of the app.
+ */
+public class LogicToDoManager extends ComponentManager implements LogicToDo {
```

Comment:
Same as above, we should probably incorporate this in the LogicManager, rather than have two managers for two separate models